### DISCUSSION TITLE
Automatic Bank retrieval Community Edition

### DISCUSSION THREAD

**Discussion by kempert** [pretix/pretix#3442] [Q&A] [answered]
Hello, ive not found in the communitiy edition the part for the automatic bank retrival. Is this possible in the version? Or is it only possbile to make in manual by .csv file?

**Comment by pc-coholic** [comment, marked as answer]
As of now, the `pretix-nordigen` plugin is only available as a paid extension covered by the pretix Enterprise agreement.

For single accounts and trusted environments, you could try using https://github.com/pretix/pretix-banktool - although this is could be a little bit more involved as it includes handling HBCI/FinTS credentials...
